sbol_utilities/component.py

```diff
@@ -621,4 +622,84 @@ def part_in_backbone(identity: str, part: sbol3.Component, backbone: sbol3.Compo
     part_in_backbone_component.features.append(backbone_subcomponent)
     # adding topology
     part_in_backbone_component.types.append(topology_type)
-    return part_in_backbone_component, part_in_backbone_seq
\ No newline at end of file
+    return part_in_backbone_component, part_in_backbone_seq
+
+def digestion(reactant:sbol3.Component, restriction_enzymes:List[sbol3.ExternallyDefined], assembly_plan:sbol3.Component)-> Tuple[sbol3.Component, sbol3.Sequence]:
+    """Digests a Component using the provided restriction enzymes and creates a product Component and a digestion Interaction.
+
```

Comment:
Note that other components may be produced too (e.g., a backbone that is being discarded), but that we are only representing the intended product.

Reply:
I added 'The product Component is assumed to be the insert for parts in backbone and the backbone for backbones.' to the docstring, that is enough to inform the user or should I specify what I look to define if it is a backbone or a part_in_backbone or that should be an unser input?

Reply:
docstring extended